packages/fhir-router/src/graphql/graphql.test.ts

```diff
@@ -686,6 +673,38 @@ describe('GraphQL', () => {
 
     const res3 = await graphqlHandler(request3, repo, fhirRouter);
     expect(res3[0]).toMatchObject(allOk);
+
+    // Fragment depth is included
+    const request4: FhirRequest = {
+      method: 'POST',
+      pathname: '/fhir/R4/$graphql',
+      query: {},
+      params: {},
+      body: {
+        query: `
+        {
+          CareTeamList {
+            ...CareTeamFields
+          }
+
+          fragment CareTeamFields {
```

Comment:
Just so I understand this test...

Inside our `MaxDepthRule` AST code, this fragment is already substituted into `CareTeamList`?  So we would be getting the "correct" depth?

Reply:
The substitution isn't done in the AST itself: there are corresponding `FragmentDefinition` and `FragmentSpread` nodes.  But out complexity calculation code handles it: https://github.com/medplum/medplum/pull/4924/files#diff-c401c28cb20cf21ea2918c9ba3d7bf86cbabb0152b53aa0e8be327e7ad80b56fR533-R546